code/modules/mob/transform_procs.dm

```diff
@@ -668,15 +669,16 @@
 	new_mob.key = key
 	new_mob.a_intent = INTENT_HARM
 
-
 	to_chat(new_mob, "You suddenly feel more... animalistic.")
 	. = new_mob
 	qdel(src)
 
 /mob/proc/Animalize()
-	var/list/mobtypes = typesof(/mob/living/simple_animal)
-	var/mobpath = input("Which type of mob should [src] turn into?", "Choose a type") in sort_list(mobtypes, GLOBAL_PROC_REF(cmp_typepaths_asc))
 
+	var/list/mobtypes = typesof(/mob/living/simple_animal)
+	var/mobpath = tgui_input_list(usr, "Which type of mob should [src] turn into?", "Choose a type", sort_list(mobtypes, /proc/cmp_typepaths_asc))
```

Comment:
This PR uses the old /proc/ reference style, update it

Reply:
fixed in 2f57bc8